nestkernel/nestmodule.cpp

```diff
@@ -510,7 +515,13 @@ NestModule::RunFunction::execute( SLIInterpreter* i ) const
    Description: sets up network calibration before run is called
    any number of times
 
-   SeeAlso: run, cleanup, simulate
+   Note: Run must only be used after Prepare is called, and
+   before Cleanup to finalize state (close files, etc).
+   Any changes made between Prepare and Cleanup may cause
+   undefined failures (the famed nose-inhabiting monkeys of
+   internet lore).
```

Comment:
That's what undefined means - up to crashes, but generally simply unexpected results (reordering of increments is the classic case).